src/blocks/recommendations/components/parsely-recommendations-inspector-controls.js

```diff
@@ -0,0 +1,183 @@
+import {
+	PanelBody,
+	PanelRow,
+	RadioControl,
+	RangeControl,
+	SelectControl,
+	TextControl,
+	ToggleControl,
+} from '@wordpress/components';
+
+const ParselyRecommendationsInspectorControls = ( {
+	attributes: { boost, imagestyle, limit, showimages, sort, tag, title },
+	setAttributes,
+} ) => {
+	return (
+		<InspectorControls>
+			<PanelBody title="Settings" initialOpen={ true }>
+				<PanelRow>
+					<TextControl
+						label={ __( 'Title', 'wp-parsely' ) }
+						value={ title }
+						onChange={ ( newval ) => setAttributes( { title: newval } ) }
+					/>
+				</PanelRow>
+				<PanelRow>
+					<RangeControl
+						label={ __( 'Maximum Results', 'wp-parsely' ) }
+						min="1"
+						max="99"
+						onChange={ ( newval ) => setAttributes( { limit: newval } ) }
+						value={ limit }
+					/>
+				</PanelRow>
+				<PanelRow>
+					<ToggleControl
+						label={ __( 'Show Images', 'wp-parsely' ) }
+						help={
+							showimages
+								? __( 'Showing images', 'wp-parsely' )
+								: __( 'Not showing images', 'wp-parsely' )
+						}
+						checked={ showimages }
+						onChange={ () => setAttributes( { showimages: ! showimages } ) }
+					/>
+				</PanelRow>
+				{ showimages && (
+					<PanelRow>
+						<RadioControl
+							label={ __( 'Image style', 'wp-parsely' ) }
+							selected={ imagestyle }
+							options={ [
+								{ label: __( 'Original image', 'wp-parsely' ), value: 'original' },
+								{ label: __( 'Thumbnail from Parse.ly', 'wp-parsely' ), value: 'thumbnail' },
+							] }
+							onChange={ ( newval ) =>
+								setAttributes( {
+									imagestyle: newval === 'original' ? 'original' : 'thumbnail',
+								} )
+							}
+						/>
+					</PanelRow>
+				) }
+				<PanelRow>
+					<TextControl
+						label={ __( 'Tag', 'wp-parsely' ) }
+						value={ tag }
+						onChange={ ( newval ) => setAttributes( { tag: newval } ) }
+					/>
+				</PanelRow>
+				<PanelRow>
+					<SelectControl
+						label={ __( 'Sort Recommendations', 'wp-parsely' ) }
+						value={ sort }
+						options={ [
+							{ label: 'Score', value: 'score' },
+							{
+								label: __( 'Publication Date', 'wp-parsely' ),
+								value: 'pub_date',
+							},
+						] }
+						onChange={ ( newval ) => setAttributes( { sort: newval } ) }
+					/>
+				</PanelRow>
+				<PanelRow>
+					<SelectControl
+						label={ __( 'Boost', 'wp-parsely' ) }
+						value={ boost }
+						options={ [
+							{
+								label: __( 'Views', 'wp-parsely' ),
+								value: 'views',
+							},
+							{
+								label: __( 'Mobile Views', 'wp-parsely' ),
+								value: 'mobile_views',
+							},
+							{
+								label: __( 'Tablet Views', 'wp-parsely' ),
+								value: 'tablet_views',
+							},
+							{
+								label: __( 'Desktop Views', 'wp-parsely' ),
+								value: 'desktop_views',
+							},
+							{
+								label: __( 'Visitors', 'wp-parsely' ),
+								value: 'visitors',
+							},
+							{
+								label: __( 'Visitors New', 'wp-parsely' ),
```

Comment:
Can we use the Description columns from https://www.parse.ly/help/api/available-metrics to populate the labels in these options? (Or at least get the labels reading better in English.)

```suggestion
								label: __( 'New Visitors', 'wp-parsely' ),
```